### ISSUE TITLE
Installation failed on ARM

### ISSUE THREAD

**Issue by PhieF** [YunoHost-Apps/akkoma_ynh#44]
Hi there

On YunoHost on raspberry pi 400 The installation of akkoma is failing

here is the complete log

https://paste.yunohost.org/raw/wegikofuje

My guess is ARM64 doesn't work with it, am I mistaking ? Thanks !

**Comment by lapineige** [comment]
We are using OTP releases instead of install from source, and [as far as I can see](https://docs.akkoma.dev/stable/installation/otp_en/#detecting-flavour) there is not ARM version.

**Comment by PhieF** [comment]
@lapineige indeed ! thanks !

**Comment by lapineige** [comment]
Here you go 😃
https://ihatebeinga.live/notice/AY7a9VwwJjfnH6RUI4

**Comment by lapineige** [comment]
I bring bad news : the latest version fo Akkoma, which introduces ARM64 builds, isn't compatible with Debian 11.
I made (an unfinished, WIP) branch to handle that ARM64 version https://github.com/YunoHost-Apps/akkoma_ynh/tree/arm64, but this will need to wait for Debian 12 compatibility in Yunohost. Sorry 😅

**Comment by lapineige** [comment]
> does this mean that ARM architecture support will need to wait until debian 12

Yes, for sure, except if we setup our own build for ARM on Debian 11.
Full disclosure : I'm not going to do it until Debian 12 compatibility. If I have the time to do it one day, it will not be before months.

> or the next Yunohost Akkoma Update will need to wait until Debian 12?

That's very likely. Except if we setup our own OTP build for AMD64, which might happens if the next update come too soon for Debian 12 and adds a too important change to wait. I don't want to do it, but if it's needed, we'll see… any contribution will be very welcomed !
